jwebunit-webdriver-plugin/pom.xml

```diff
@@ -9,17 +29,7 @@
     <artifactId>jwebunit-webdriver-plugin</artifactId>
     <name>Webdriver Plugin</name>
     <description>Webdriver plugin for JWebUnit.</description>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <testFailureIgnore>true</testFailureIgnore>
```

Comment:
Indeed, ignoring test failures is not a good practice and it would be better to have a dedicated reporting of unsupported cases. But as you probably noticed I have stopped any dev effort on JWebUnit so don't wait for me on this :)